Anzee/APIErrorResponse.swift

```diff
@@ -9,8 +9,8 @@
 import Foundation
 
 /// Basic struct to parse any possible API error responses
-struct APIErrorResponse: Codable {
-    let status: Int
-    let type: String
-    let detail: String
+public struct APIErrorResponse: Codable {
+    public let status: Int
```

Comment:
If the top level structure is `public` do all the child elements need to be?

Reply:
Apparently not. Fixed. :)

Reply:
Ok, so Jennifer reminded me the reason these need to be `public` is if they are accessed outside the SDK.